composer.json

```diff
@@ -29,7 +29,7 @@
         "php": "^7.1",
         "squizlabs/php_codesniffer": "^3.3.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.4.2",
-        "slevomat/coding-standard": "^4.7.0"
+        "slevomat/coding-standard": "^4.7.2"
```

Comment:
This change is unrelated (to be handled in #77), please drop it.

Reply:
Also in case we would need to revert this sniff, we would also revert version bump -> not acceptable as one commit.